File: example/09dot4/main.m
syms sigma;
right=2*sigma.^3./((1+sigma.*sqrt(sigma.^2+4)).*sqrt(4.*sigma.^2+(sigma.^2+1).^2));
s=latex(right);
sigma=0:0.01:10;
right=2*sigma.^3./((1+sigma.*sqrt(sigma.^2+4)).*sqrt(4.*sigma.^2+(sigma.^2+1).^2));
h=figure(1);hold on;
[m,i]=max(right);
plot(sigma,right,"-",sigma(i),m,"o")
xlabel("\sigma");
ylabel("RHS");
text(4,0.17,"$"+s+"$","Interpreter","latex","FontSize",18)
text(sigma(i)+0.5,m,sprintf("(%0.2f,%.4f)",sigma(i),m))
set(h,'PaperSize',[20 10]); %set the paper size to what you want  
print(h,'example_9dot4','-dpdf',"-fillpage") % then print it

File: example/09dot4/main.py
# plot the RHS for Delta_m
#%%
import numpy as np
import matplotlib.pyplot as plt

# %%
from sympy.abc import x,sigma
from sympy import sqrt,latex
num=2*sigma**3
den1=1+sigma*sqrt(sigma**2+4)
den2=sqrt(4*sigma**2+(sigma**2+1)**2)
rhs=num/den1/den2

x=np.linspace(0,10,1000)
y=np.array(list(map(lambda x: rhs.subs(sigma,x),x)))
max_index=np.argmax(y)

# %%
plt.figure(figsize=(8,4))
plt.plot(x,y,"b-")
plt.plot(x[max_index],y[max_index],"ro")
plt.text(x[max_index],y[max_index]+0.01,"({:.4f},{:.4f})".format(x[max_index],y[max_index]))
plt.text(4,0.17,'$'+latex(rhs)+'$',fontsize=20,color="blue")
plt.grid()
plt.xlabel(r'$\sigma$')
plt.ylabel(r'$RHS$')
plt.ylim(0,0.45)
plt.savefig("rhs.pdf",transparent=True)
# %%
